administrator/language/en-GB/lib_joomla.ini

```
@@ -376,8 +376,11 @@ JLIB_HTML_BATCH_MOVE="Move"
 JLIB_HTML_BATCH_MOVE_QUESTION="Action to Perform"
 JLIB_HTML_BATCH_NO_CATEGORY="- Don't copy or move -"
 JLIB_HTML_BATCH_NOCHANGE="- Keep original Access Levels -"
-JLIB_HTML_BATCH_TAG_LABEL="Add Tag"
-JLIB_HTML_BATCH_TAG_NOCHANGE="- Keep original Tags -"
+JLIB_HTML_BATCH_TAG_MENU_LABEL="Add or remove a Tag"
+JLIB_HTML_BATCH_TAG_NO_CHANGE="- Don't add or remove Tags -"
+JLIB_HTML_BATCH_TAG_ADDREMOVE_QUESTION="Action to Perform"
+JLIB_HTML_BATCH_TAG_ADD="Add"
+JLIB_HTML_BATCH_TAG_REMOVE="Remove"
```

```suggestion
JLIB_HTML_BATCH_TAG_ADD="Add"
JLIB_HTML_BATCH_TAG_ADDREMOVE_QUESTION="Action to Perform"
JLIB_HTML_BATCH_TAG_MENU_LABEL="Add or remove a Tag"
JLIB_HTML_BATCH_TAG_NO_CHANGE="- Don't add or remove Tags -"
JLIB_HTML_BATCH_TAG_REMOVE="Remove"
```
Alpha ordering language strings.